npm-postinstall.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const submodules = [
+  'project',
```

Comment:
Extract that lsit into a config file

Reply:
@ridz1208 @driusan @regisoc  Do you have any suggestion of where that list of paths should go?

Reply:
No. I'm not even sure it needs to be, personally.. when do we have situations where it will need to be updated that isn't in the project directory and isn't a module committed to the repo?

Reply:
Agree, let's keep it there for now.